### DISCUSSION TITLE
Raise the limit of Virtual Cards from 5.

### DISCUSSION THREAD

**Discussion by stevebissett** [Investec-Developer-Community/.github#20] [Ideas]
There was a lot of discussion at the Cape Town meetup (Monday 27 May, 2024) about the limit of 5 virtual cards and the need for these to be recycled. 

Rudi Storm mentioned this as a limitation in his presentation.

A few points were raised: 
- A virtual card must be linked to a natural person. 
- Cards cannot currently be deleted once created (some limitations on deleting cards were mentioned to prevent deletion straight after transactions are made).

I would propose that the limit be raised to >= 50 virtual cards, and adding a mechanism to safely discontinue/cancel virtual cards after their intended lifespan. 

For reference - currently using Spendesk.com for virtual cards for business purposes:
https://www.youtube.com/watch?v=d2g9dInja2A
https://helpcenter.spendesk.com/en/articles/4046280-request-a-virtual-card
https://www.spendesk.com/product/virtual-cards/

**Comment by adejongh** [comment]
The default could definitely be more, and if a fixed max is not an option another way would be to just contact your banker to have it increased for you.

**Comment by neilwithers** [comment]
Hi Steve, we are going to close this for now. If the limit changes, we will let you know.
